### ISSUE TITLE
Add Item Set List Not Updating

### ISSUE THREAD

**Issue by YarbySauce** [thommcgrath/Beacon#111]
The "Add Item Set" List under Document is not updating & is empty. 

Would it be possible to have a scroll-able drop down when adding an existing preset? Maybe also add a list for your Custom presets Grouping categories so you don't have to type them out each time.

Thanks.

**Comment by thommcgrath** [comment]
The first issue is a bug. It’ll be fixed in the next update. Until then, you can click and hold the “+” button in the item sets column to show the presets menu.

I don’t know what you mean by “scrollable drop down when adding an existing preset” though. So if you can clarify, that would be great.

The grouping menu isn’t a bad idea, but the ComboBox control that my dev environment uses is better left unused. I’ll see if I can get autocomplete working or something instead.

**Comment by YarbySauce** [comment]
When I use the + it shows the whole list on screen & some items get clipped at the bottom.

**Comment by thommcgrath** [comment]
Yeah. Like every other menu, just bring your cursor towards the bottom. It scrolls.

**Comment by YarbySauce** [comment]
I figured it out, I like my taskbar on top. When I moved it to the bottom, I could use the arrows to scroll.
